sys/sys_uartprint.c

```diff
@@ -0,0 +1,14 @@
+#include <syscall.h>
+#include <stdc.h>
+
+void sys_uart_print_char(syscall_args_t *sa) {
+  char c = sa->args[0];
+  kprintf("%c", c);
+}
+
+void sys_uart_print_str(syscall_args_t *sa) {
+  /* TODO: Copy string from user space? Is that even necessary for
+     this case? */
+  char *c = (char *)sa->args[0];
+  kprintf("%s", c);
```

Comment:
Passing unbounded buffer to kernel is evil!

Please reimplement both syscalls in terms of SYS_write with file descriptor no. 1.